crates/burn-candle/src/lib.rs

```diff
@@ -159,6 +160,7 @@ mod tests {
     burn_autodiff::testgen_ad_mul!();
     burn_autodiff::testgen_ad_neg!();
     burn_autodiff::testgen_ad_recip!();
+    burn_autodiff::testgen_ad_remainder!();
```

Comment:
If the autodiff remainder also fails on macos then we could also comment this test out with a note.